src/tracker/WorklogsToSave/index.js

```diff
@@ -61,22 +62,28 @@ const WorklogsToSave = ({ handleSubmit, switchMode }: Props) => (
         CANCEL
       </Link>
     </header>
-    <PointsByIssue />
-    <Fab
-      autoFocus
-      color="primary"
-      aria-label="Submit worklogs"
-      className="fab"
-      onClick={handleSubmit}
-    >
-      <CheckCircleOutlineIcon />
-    </Fab>
+    {issues.length < 1 ? (
+      <Loading />
+    ) : (
+      <React.Fragment>
+        <PointsByIssue />
+        <Fab
+          autoFocus
+          color="primary"
+          aria-label="Submit worklogs"
+          className="fab"
+          onClick={handleSubmit}
+        >
+          <CheckCircleOutlineIcon />
+        </Fab>
+      </React.Fragment>
+    )}
   </div>
 )
 
 export default compose(
   connect(
-    null,
+    state => ({ issues: state.issues }),
     { submitWorklogs: _submitWorklogs, switchMode: _switchMode }
   ),
```

Comment:
you can write here:
```suggestion
  branch(
    ({ issues }) => !issues.length,
    renderComponent(Loading)
  )
```
and avoid this logic in the component itself :) imo it's nicer this way

Reply:
Using branch is nice, but look kinda weird to me, when shortcuts panel moves so much.